content/issue-2/applicative-parsing.md

```diff
@@ -0,0 +1,716 @@
+# Programming is hard, looking for better abstractions
+
+If you ever wrote a program bigger than a few lines of code you probably know that the bigger
+the program gets - the harder it's to make sure it does exactly what you meant due to how
+different parts can interact. Over time programmers and computer scientists discovered multiple
+ways to manage this complexity. Programming went from physically wiring the instructions to
+writing them as mnemonics with assembly language, to abstractions with variables, labels,
+functions, and so on.
+
+[Functional Programming](https://en.wikipedia.org/wiki/Functional_programming)
+is one of the paradigms for managing complexity with its main idea of writing code as a set of
+functions and using those functions to pass values around instead imperative style of steps
+to perform to mutate some data.
```

Comment:
```suggestion
[Functional Programming](https://en.wikipedia.org/wiki/Functional_programming)
is one of the paradigms for managing complexity. Its main idea is to abstract code as a set of
functions, and using those functions to pass values around rather than imperatively
to mutate data.
```